river/naive_bayes/complement.py

```diff
@@ -12,7 +15,8 @@
 class ComplementNB(base.BaseNB):
     """Naive Bayes classifier for multinomial models.
 
-    The input vector has to contain positive values, such as counts or TF-IDF values.
+    This estimator supports learning with mini-batches. The input vector has to contain positive
+    values, such as counts or TF-IDF values.
```

Comment:
Maybe you want to mention that said counts and TF-IDF values can be obtained via `BagOfWords` and `TFIDF`.